shell/edit/logging-flow/Match.vue

```diff
@@ -51,19 +58,22 @@ export default {
 
 <template>
   <div>
-    <KeyValue
-      v-model:value="value.labels"
-      :title="value.select ? t('logging.flow.matches.pods.title.include') : t('logging.flow.matches.pods.title.exclude')"
-      :mode="mode"
-      :initial-empty-row="true"
-      :read-allowed="false"
-      :title-add="true"
-      protip=""
-      :key-label="t('logging.flow.matches.pods.keyLabel')"
-      :value-label="t('logging.flow.matches.pods.valueLabel')"
-      :add-label="t('logging.flow.matches.pods.addLabel')"
-    />
-    <div class="spacer" />
+    <div v-if="!isHarvester">
```

Comment:
I think we'd prefer pushing the v-if onto KeyValue to avoid adding another div wrapper

Reply:
I'll use a <template>, because the v-if needs to include the `<div class="spacer" />` below the <KeyValue>